### ISSUE TITLE
Add option to limit number of publications

### ISSUE THREAD

**Issue by jkeirstead** [jkeirstead/scholar#11]
In the examples, `compare_scholars` and `compare_scholar_careers` ([src](R/compare.r)) takes ages to run because it's pulling down hundreds of publications for Stephen Hawking (damn those prolific scientists).  I've changed this to Isaac Newton for the moment and made the slowest `compare_scholars` example __DONTRUN__ but that's a rather temporary fix.

**Comment by jefferis** [comment]
See also #15 where the increase in pagesize from 20 to 100 changes this example:

```r
 ## How do Richard Feynmann and Isaac Newton compare?
	ids <- c("B7vSqZsAAAAJ", "xJaxiEEAAAAJ")
    df <- compare_scholars(ids)
```

from 20 to 6 s for me.

**Comment by jkeirstead** [comment]
Fixed with bafe993bbc44ae08bea5fd01bd08e193d337408e
